### ISSUE TITLE
Standard tags

### ISSUE THREAD

**Issue by loolmeh** [Tatoeba/tatoeba2#79]
Display list of common standard tags, so that users can tag a sentence just with one (or a few) click(s).

---

Original ticket: https://www.assembla.com/spaces/tatoeba2/tickets/285

**Comment by agrodet** [comment]
Anyone could give more context or details to this unclear topic?

**Comment by jiru** [comment]
A bit of history:

1. This ticket was created by Trang in 2010, when tags were initially implemented. At this time, Tatoeba was not on Github, instead it used Assembla. They created a bunch of tickets related to the implementation of tags, but eventually some of them, like this one, were left undone.
3. In 2014, the tickets were migrated to Github using loolmeh’s account, so he has little to do with this even though he opened this ticket.

While we have no such thing as standard tags (except @ ones), I think the description is pretty clear in terms of UI: allows to quickly tag a sentence from a list of proposed tags. However there is no such list because tags are not standardized. There were some [efforts to tidy up tags after their first implementation](https://blog.tatoeba.org/2010/11/tags-guidelines.html), but nothing went as far as standardization, just guidelines.

Nevertheless I think the idea is worth exploring because it could help people tagging, and better tagging would help categorization, and better categorization would increase the value of the corpus.

What could be the list of suggested tags? Should we set up a list of standard tags?

**Comment by alanfgh** [comment]
Well, three of the tags that are actually mentioned and linked to in our UI certainly deserve a special place. These are \@change, \@check, and \@needs native check, all of which are visible from "Contribute -> Improve sentences -> Tags you should know about". (\@OK is also visible from that page, but it serves a different purpose and, I think, should not be mentioned there, though I realize that's a different subject.)

There are various tags whose names begin with \@ that are used in the same way as \@change, \@check, and \@needs native check (that is, to mark sentences that need further action) but are not mentioned on that page. One would certainly be \@delete. There are others that are similar, though they're not as canonical: \@delete maybe, \@please adopt, \@NNC, \@check translation, \@change grammar, \@possible copyright infringement, \@change punctuation, \@check copyright, \@change or delete, \@change or unlink, \@check link, \@unlink, @change audio. There is also "\@not a sentence" (which probably serves only as a warning marker; it's quite likely no one is using it to find sentences to delete or modify). Then there are a few one-off tags. You can see all tags starting with "\@" by choosing "Browse by tag", then typing "@" into the "Search tags" field and clicking the "SEARCH" button.

**Comment by jiru** [comment]
Thanks for the summary about tags starting with @, Alan. These tags describe what to do with the sentence. I was actually more thinking about tags that describe what the sentence is (that is to say pretty much anything not starting with @).

**Comment by agrodet** [comment]
Thank you for the explanation, it will be useful for my summary. I think it is difficult to provide tags that are "generally" widely used. Different people will tag different things, using most commonly used would skew the results towards people that tag massively (and possibly biases the choice of tag). Right now, the best choice I came up with is to display two categories: most recently used tags (like the "add to list" function AND most useful utility tags. Obviously, regular users wouldn't have the second category. 

Of course we would need to rework how we add tags, and I already summarized my idea(s), so pardon me for not going through the details here. I will post my summary today or tomorrow.
